test/lazy-loading.test.js

```diff
@@ -123,6 +123,22 @@ describe('lazy loading', () => {
         expect(comboBox.selectedItem).to.be.null;
       });
 
+      describe('when autoOpenDisabled', () => {
+        beforeEach(() => (comboBox.autoOpenDisabled = true));
+
+        (isComboBoxLight ? it.skip : it)('should not clear filter when loading first page', () => {
+          expect(comboBox.autoOpenDisabled).to.be.true;
+          expect(comboBox.opened).to.be.false;
+          comboBox.dataProvider = spyDataProvider;
+          comboBox.filter = 'item 1';
```

Comment:
Let's mimic the user input instead to emulate setting `filter`:

```suggestion
          setInputValue('item 1');
          comboBox.inputElement.dispatchEvent(new CustomEvent('input'));
```